src/rules/javascript/eval_string_arg.js

```diff
@@ -0,0 +1,31 @@
+import { EVAL_STRING_ARG } from 'messages/javascript';
+import { getVariable } from 'utils';
+
+export default function(context) {
+  return {
+    CallExpression: function(node) {
+      var acceptedArgs = ['FunctionExpression', 'ArrowFunctionExpression'];
+
+      if (node.callee.property.type === 'Identifier' &&
+          (node.callee.property.name === 'setTimeout' ||
+           node.callee.property.name === 'setInterval')) {
+
+        if (node.arguments.length > 0 &&
+            node.arguments[0].type === 'Identifier') {
+          var originalDef = getVariable(context, node.arguments[0].name);
+          if (originalDef !== undefined &&
```

Comment:
I guess this is ok - I've tended to use `typeof originalDef !== 'undefined'` old habit from when undefined could get redefined. Old habits die-hard.

We should decide on which way we want to go with this and make jscs enforce it either way. I like consistency more than anything else.

<img alt="1__node" src="https://cloud.githubusercontent.com/assets/1514/10842537/3b10380e-7eea-11e5-9cbe-a9dd7dd6dd06.png">